docs/how-to-guides/develop-a-tool/create-custom-llm-tool.md

```diff
@@ -0,0 +1,62 @@
+# Create Custom LLM Tool
+In this document, we will guide you through the process of developing a tool with `custom_llm` type which can connect to a customized large language model with prompt.
+
+A Custom LLM is a large language model fine-tuned by yourself. If you find it has good performance and is useful, then you can follow this guidance to make it a tool so that it can be shared with others to increase its impact.
```

Comment:
You are misunderstood by custom_llm type meaning. It is not user fine-tuned model.

Reply:
Fixed, thanks.